api/envoy/config/core/v3/protocol.proto

```diff
@@ -149,8 +149,9 @@ message HttpProtocolOptions {
 
   // The maximum duration of a connection. The duration is defined as a period since a connection
   // was established. If not set, there is no max duration. When max_connection_duration is reached
-  // the connection will be closed. Drain sequence will occur prior to closing the connection if
-  // if's applicable. See :ref:`drain_timeout
+  // the connection will be closed. Any inflight requests will still get processed correctly if this
+  // timeout fires during request processing. Drain sequence will occur prior to closing the connection
+  // if it's applicable. See :ref:`drain_timeout
```

Comment:
I think it's not quite accurate to say that all in flight requests will be processed.
Maybe rephrase here and below to make it clear that if there's no active streams, the connection will be closed and if there are active streams, the drain sequence will kick off and the connection will be force-closed after the drain period?

Reply:
Thanks, @alyssawilk. I rephrased it as per your suggestions.